_format/1.1/index.md

```diff
@@ -504,18 +513,22 @@ For example:
 ### <a href="#document-links" id="document-links" class="headerlink"></a> Links
 
 Where specified, a `links` member can be used to represent links. The value
-of each `links` member **MUST** be an object (a "links object").
+of this member **MUST** be an object (a "links object").
 
-Each member of a links object is a "link". A link **MUST** be represented as
-either:
+<a href="#document-links-link" id="document-links-link"></a>
+Within this object, a link **MUST** be represented as either:
```

Comment:
I agree that there's an analogy between the text here and #946. 

I faced a couple problems writing this section:

1. With the profile key now holding multiple links, we ca no longer say "each value is a link".
2.  Because we haven't yet defined the relationship between `links` and RFC 5988, I didn't want to introduces a term for the key (i.e. I didn't want to say whether it's the links name or a relation).

So, given the above constraints, the wording in the PR was what I came up with.

Let's get this PR merged, and then we can figure out how to clarify things in #958. I'll link this thread there for reference.